src/index.js

```diff
@@ -85,8 +88,16 @@ class QrCodeScanner extends Component {
     qr.decode(imageData);
   }
 
+  bindVideoStream() {
+    if (typeof this.videoTag !== 'undefined') {
+      return (this.videoTag.srcObject = this.stream);
```

Comment:
It's bad practice to return an assignment because it's difficult to tell the intent of the `return` statement here.

```suggestion
      this.videoTag.srcObject = this.stream
      return;
```